guides/v2.1/coding-standards/docblock-standard-javascript.md

```diff
@@ -104,9 +106,9 @@ function&nbsp;someMethod(obj)&nbsp;{
 project.MyClass.prototype.someProperty&nbsp;=&nbsp;4;
 </pre>
 
-<h3 id="frontend-dev-guide-javascript-js-coding-docblock-tagreference">JSDoc tag reference</h3>
+### JSDoc tag reference
```

Comment:
Please append `{#frontend-dev-guide-javascript-js-coding-docblock-tagreference}` to preserve the custom id.